src/explore-education-statistics-admin/src/types/ckeditor.ts

```diff
@@ -34,6 +34,15 @@ export interface AutoSavePluginConfig {
   waitingTime: number;
 }
 
+export interface AlignmentFormat {
+  className?: string;
+  name: 'left' | 'right' | 'center' | 'justify';
+}
+
+export interface AlignmentConfig {
+  options: Array<AlignmentFormat['name'] | AlignmentFormat>;
```

Comment:
Small nitpick: We typically specify arrays with the `[]` syntax instead of using `Array`